master/docs/developer/www.rst

```diff
@@ -617,6 +617,139 @@ Several methods are added to each "restangularized" objects, aside from get(), p
     Call the control data api.
     This builds up a POST with jsonapi encoded parameters
 
+DataService
+.............
+
+DataService is the future replacement of BuildbotService for accessing the Buildbot data API.
+It has a modernized interface for accessing data. It uses IndexedDB for storing cached data as a single data store,
+and LocalStorage for broadcasting events between browser tabs. DataService works in a master/slave architecture.
+The master browser tab is responsible for keeping the requested data up to date in the IndexedDB and notify slaves when a data is ready to be used or it is updated.
+It handles both the Rest API calls and the WebSocket subscriptions globally.
+
+It uses the following libraries:
+
+* Dexie.js (https://github.com/dfahlander/Dexie.js) - Minimalistic IndexedDB API with bulletproof transactions
+* Tabex (https://github.com/nodeca/tabex) - Master election and in browser message bus
+
+The DataService is available as a standalone AngularJS module.
+Installation via bower:
+
+  .. code-block:: none
+
+      bower install buildbot-data --save
+
+Inject the ``bbData`` module to your application:
+
+  .. code-block:: javascript
+
+      angular.module('myApp', ['bbData'])
```

Comment:
It's not that significant, I can change this. Everyone use abbreviated module names so I thought that I do too.